pubspec.yaml

```diff
@@ -91,24 +91,43 @@ flutter:
     - assets/reset/
     - assets/non-free/
     - assets/disclosure/
+    - assets/ui/
   fonts:
-    - family: Open Sans
+    - family: Alexandria
       fonts:
-        - asset: fonts/open-sans/OpenSans-Regular.ttf
-          weight: 400
-        - asset: fonts/open-sans/OpenSans-Medium.ttf
-          weight: 500
-        - asset: fonts/open-sans/OpenSans-MediumItalic.ttf
-          weight: 500
-          style: italic
-        - asset: fonts/open-sans/OpenSans-Bold.ttf
-          weight: 700
-        - asset: fonts/open-sans/OpenSans-BoldItalic.ttf
-          weight: 700
-          style: italic
-    - family: IrmaIcons
```

Comment:
The icons on the more tab and the settings screen are missing now due to the removal of `IrmaIcons`. Was this planned?